blt/src/Hooks/HumsciHook.php

```diff
@@ -16,6 +16,11 @@ class HumsciHook extends BltTasks {
    * @hook post-command drupal:sync
    */
   public function postDrupalSync($result, CommandData $commandData) {
+    $this->taskDrush()
```

Comment:
What? No local simplesaml? I think you could have this setup and available from within the repo. I suppose that is out of scope tho.